cmake/SanitizerHelper.cmake

```diff
@@ -33,6 +33,9 @@ if(NRN_SANITIZERS)
   if("address" IN_LIST nrn_sanitizers)
     list(APPEND NRN_COMPILE_DEFS NRN_ASAN_ENABLED)
   endif()
+  if("thread" IN_LIST nrn_sanitizers)
+    list(APPEND NRN_COMPILE_DEFS NRN_TSAN_ENABLED)
+  endif()
```

Comment:
I think it would be a good idea to update docs/cmake_doc/options.rst and docs/install/debug.md to say one can use -DNRN_SANITIZERS=thread

Reply:
OK, done in f7efe6d19a2181f5e797964417547a0cb1f16a46.